spec/system/users_spec.rb

```diff
@@ -0,0 +1,300 @@
+require 'rails_helper'
+
+describe 'user registration' do
+  it 'visit register page' do
+    visit root_path
+    assert_text 'Code Play'
+    assert_text 'Entrar'
+    assert_text 'Cadastrar'
+
+    click_on 'Cadastrar'
+    assert_current_path new_user_registration_path
+
+    assert_no_text 'Cadastrar'
```

Comment:
O que acham de usar a sintaxe do RSpec nesses testes? Vou deixar um exemplo

```suggestion
    expect(current_path).to eq(new_user_registration_path)

    expect(page).not_to have_text('Cadastrar')
```

Reply:
Alterei todos os `assert_current_path`, `assert_text` e `assert_no_text` para a sintaxe do RSpec.